packages/teams-js/src/public/dialog.ts

```diff
@@ -33,6 +33,15 @@ export namespace dialog {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   export type PostMessageChannel = (message: any) => void;
   export type DialogSubmitHandler = (result: ISdkResponse) => void;
+  const storedMessages: string[] = [];
+
+  export function initialize(): void {
```

Comment:
We should probably hide this from the documentation since I assume we don't want users to call this

Reply:
Good idea, I'll add in the correct tag